build.bat

```diff
@@ -0,0 +1,29 @@
+WHERE choco
+IF %ERRORLEVEL% NEQ 0 %PS% -NoProfile -ExecutionPolicy unrestricted -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET PATH=%PATH%;%systemdrive%\ProgramData\chocolatey\bin
+choco install sbt --acceptlicense -y
+WHERE sbt
+IF %ERRORLEVEL% NEQ 0 SET PATH=%PATH%;%systemdrive%\Program Files (x86)\sbt\bin
+choco install "\\columbus\sw_share\DEPARTMENT\SD Software\Java\jdk8.8.0.66.nupkg" --acceptlicense -y
```

Comment:
Makes sense - we'll get build.bat out of the repo. 

BTW, the reason we had to download the jdk nuget in a local drive is that Chocolatey install from the Oracle site doesn't work due to cert issues.

I'll submit a check-in for this and other comments above this afternoon.

@all: Thanks for looking at this PR.